packages/utils/src/__tests__/object.test.js

```diff
@@ -263,23 +270,24 @@ describe('object', () => {
           code: 'FJ',
         },
       ],
-    ])('%s', (_, objectCollection, expected) => {
+    ];
+
+    it.each(testData)('%s', (_, objectCollection, expected) => {
       expect(flattenToObject(objectCollection)).toStrictEqual(expected);
     });
   });
 
   describe('reduceToSet()', () => {
-    const expectedResult = new Set(['id1', 'id2']);
-    it.each([
-      ['should create a set out of an array of objects', [object1, object2], 'id', expectedResult],
-      [
-        'should create a set out of an object dictionary',
-        { id1: object1, id2: object2 },
-        'id',
-        expectedResult,
-      ],
-    ])('%s', (_, objectCollection, property, expected) => {
-      expect(reduceToSet(objectCollection, property)).toStrictEqual(expected);
+    const object1 = { id: 'id1', value: 10 };
+    const object2 = { id: 'id2', value: 20 };
```

Comment:
We use the same `object1`, `object2` variables in lots of test suites here, but it's a long file and those are simple values so it's better to just redefine them in this case

At some point we could break this file into smaller ones, as you will see I did with `period.test.js` in later commits